components/registry-facade/cmd/run.go

```diff
@@ -56,19 +56,28 @@ var runCmd = &cobra.Command{
 			log.WithField("fn", authCfg).Info("using authentication for backing registries")
 		}
 
+		reg := prometheus.NewRegistry()
+		rtt, err := registry.NewMeasuringRegistryRoundTripper(http.DefaultTransport, prometheus.WrapRegistererWithPrefix("downstream_", reg))
```

Comment:
Any reason why gitpod's metrics doesn't start with `gitpod_`?

Reply:
As seen at prometheus docs:
```
The prefix is sometimes referred to as namespace by client libraries. For metrics specific to an application, the prefix is usually the application name itself.
```

We can also see a nice example at prometheus go-client:

https://github.com/prometheus/client_golang/blob/39b478e90c0bbd869817725d4553687513b7cf01/prometheus/examples_test.go#L36-L42